tools/c7n_mailer/requirements.txt

```diff
@@ -8,6 +8,6 @@ jsonschema>=2.5.1
 ruamel.yaml>=0.15.93
 six>=1.12.0
 datadog==0.28.0
-sendgrid==5.6.0
+sendgrid==6.0.5
```

Comment:
needs corresponding mailer changes per https://github.com/sendgrid/sendgrid-python/blob/master/use_cases/README.md

Reply:
Have to remember to change this in mailer setup.py, that is the one users get.

My PR has all the fixes for the new SDK so that should help.

#4028

Reply:
awesome, thanks